config/locales/ja.yml

```
@@ -355,7 +355,7 @@ ja:
         mail: メーリングリスト
         report_abuse: 悪用報告
         reverse_dependencies: 被依存性
-        review_changes:
+        review_changes: 変更点を確認
```

Hi @ybiquitous. I think `差分` (diff) is simple word and straightforward, but what about?
The link destination is also "**Diff**end" :-) https://my.diffend.io/

```suggestion
        review_changes: 差分
```

---

Nice idea! 👍🏼